configure.ac

```diff
@@ -3425,6 +3425,20 @@ case "$ac_sys_system" in
 esac
 AC_MSG_RESULT($SHLIBS)
 
+dnl perf trampoline is Linux specific and requires an arch-specific
```

Comment:
What is Linux specific about it?  Anything could use this feature as all sorts of systems as most platforms have tooling that collects C stack information. It is worth keeping the reasoning documented explicitly in code comments.

Reply:
> What is Linux specific about it?

The whole infrastructure is for perf and that's Linux specific. The map for the trampolines is written specifically with perf format in the file location asked by perf. I dont see how having an intermediate frame without proper information could help other systems or tools